### ISSUE TITLE
Add a new Telemetry uptake information about the number of people having the update with regards to the number of users of the previous version

### ISSUE THREAD

**Issue by Natim** [mozilla/PollBot#171]
Add a new Telemetry uptake information about the number of people having the update with regards to the number of users of the previous version

https://sql.telemetry.mozilla.org/queries/50383/source

**Comment by Natim** [comment]
```

WITH updated_t AS (
    SELECT COUNT(*) AS updated
    FROM telemetry_update_parquet
    WHERE submission_date_s3 >= '201712'
      AND metadata.normalized_channel = 'aurora'
      AND payload.reason = 'success'
      AND environment.build.build_id IN ('20180122144853')
),
total_t AS (
    SELECT COUNT(*) AS total, metadata.normalized_channel AS version
    FROM telemetry_update_parquet
    WHERE submission_date_s3 >= '201712'
      AND metadata.normalized_channel = 'aurora'
      AND payload.reason = 'success'
      AND environment.build.build_id IN ('20180122144853')
      GROUP BY 2
)
SELECT updated * 1.0 / total as ratio, updated, total, version
FROM updated_t, total_t
```

**Comment by Natim** [comment]
```
echo '{
    "aggs": {
        "by_version": {
            "aggs": {
                "build_ids": {
                    "terms": {
                        "field": "target.version"
                    }
                }
            },
            "terms": {
                "field": "build.id",
                "order": {
                    "_term": "desc"
                },
                "size": 5   
            }
        }
    },
    "query": {
        "bool": {
            "filter": [
                {
                    "term": {
                        "source.product": "firefox"
                    }
                },
                {
                    "term": {
                        "target.channel": "beta"
                    }
                }
            ]
        }
    },
    "size": 0
}
' | http POST https://buildhub.prod.mozaws.net/v1/buckets/build-hub/collections/releases/search
```

**Comment by Natim** [comment]
Implemented in #181 but it doesn't seems to be interesting enough. I am going to implement #172 instead.
